### ISSUE TITLE
Investigate removal of explicit schema names from generated SQL on Postgres

### ISSUE THREAD

**Issue by tomhoule** [prisma/prisma-engines#1131]
TL;DR: `CREATE TABLE "public"."Users" (...);` -> `CREATE TABLE "Users" (...);`.

This could be a source of bugs/footguns, if users connect to a database with a different schema in production and development in particular. Potential generated raw SQL would include the schema name, and not work on the production database.

We should:

1. [ ] confirm whether this is at all possible (some statements may require qualified names). We can test this by removing all places where we use the schema in `postgres_renderer.rs`.
2. [ ] decide whether we want to do this. The schema is part of the database.
3. [ ] do it

internal issue: https://github.com/prisma/migrations-team/issues/24
